java/RSPEC-1845.java

```diff
@@ -0,0 +1,7 @@
+class CaseDiff1 {
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._